iOS/PixelDefinitions/pixels/TEMPLATE.json5

```diff
@@ -0,0 +1,28 @@
+// See https://app.asana.com/0/72649045549333/1208001118995887/f.
+{
+    pixel_name: {
+        description: 'Describe when the pixel fires and its purpose',
+        // DDG usernames of the people responsible for the pixel.
```

Comment:
Should update this to github usernames as we are in the process of switching to that